src/ilasm/main.cpp

```diff
@@ -171,7 +171,7 @@ extern "C" int _cdecl wmain(int argc, __in WCHAR **argv)
       printf("\n/CLOCK          Measure and report compilation times");
 //      printf("\n/ERROR          Try to create .exe or .dll file despite errors reported");
 //      printf("\n       Warning! Results are unpredictable, use this option at your own risk!");
-      printf("\n/RESOURCE=<res_file>    Link the specified resource file (*.res) \n\t\t\tinto resulting .exe or .dll");
+      printf("\n/RESOURCE=<obj_file>    Link the specified object file (*.obj) \n\t\t\tinto resulting .exe or .dll");
```

Comment:
I do not think we want to be moving towards documenting this. This cannot be just any random object file. It has to be a very particular obj file with particular names of sections, particular format of relocs, etc. The only tool that can ever generate the file in this format is cvtres.exe with the specific set of command line switches.